projects/igniteui-angular/src/lib/grids/columns/interfaces.ts

```diff
@@ -51,6 +51,13 @@ export interface IFieldPipeArgs {
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface IColumnPipeArgs extends IFieldPipeArgs {}
 
+export interface IFieldEditorOptions {
+    /** A custom or a predefined format string used for the built-in editors of date/time columns */
```

Comment:
```suggestion
    /** A custom input format string used for the built-in editors of date/time columns */
```
And here is where we could drop a link. With API I'd love to be able to link to either the directive or time picker `inputFormat` option, but none of those explain their supported formats. So the best option it to explain this sets the `inputFormat` of the editor + see the Editing section under https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/column-types#datetime-date-and-time